apps/wallet-mobile/src/features/Swap/common/AmountCard/AmountCard.tsx

```diff
@@ -60,7 +60,7 @@ export const AmountCard = ({
         {label != null && <Text style={[styles.label, !isEmptyString(error) && styles.labelError]}>{label}</Text>}
 
```

Comment:
maybe, (p.s dunno if the exactly same thing)
```
const handleOnPress = () => noTokenSelected ? navigateTo() : focusInput()
```

Reply:
Would do the same thing

Reply:
if it is the same thing I'd say to implement as handle.. become a little more readable.